packages/ag-charts-community/src/chart/legend.ts

```diff
@@ -889,11 +888,10 @@ export class Legend {
         const calculateTranslationPerpendicularDimension = () => {
             switch (this.position) {
                 case 'top':
+                case 'left':
```

Comment:
I'm not entirely sure what it'll complain about. 
These two values simply return the same result logically.
How does it differ from the rest of the code on the same file? 
e.g.
```
            switch (this.position) {
                case 'top':
                case 'bottom':
                    translationX = (shrinkRect.width - legendBBox.width) / 2;
                    translationY = calculateTranslationPerpendicularDimension();
                    newShrinkRect.shrink(legendBBox.height, this.position);
                    break;

                case 'left':
                case 'right':
                default:
                    translationX = calculateTranslationPerpendicularDimension();
                    translationY = (shrinkRect.height - legendBBox.height) / 2;
                    newShrinkRect.shrink(legendBBox.width, this.position);
            }
```

Reply:
Let's roll with it, I can't find the rule I'm thinking of in the current active SonarCloud ruleset, let's just deal with anything (or not) the report flags up - this is an example of what it has required previously:
https://github.com/ag-grid/ag-charts/blob/ce01f3cdeadc6bcd1d0a258c8b5f23df84413ef4/packages/ag-charts-community/src/scale/timeScale.ts#L154

...but maybe that is a different pattern since each clause is actually doing something.